script.js

```diff
@@ -135,11 +135,27 @@ function initListeners() {
     if (target.classList.contains("options__trash")) {
       let str = target.closest(".notes__item");
       str = str.querySelector(".notes__descr").textContent;
-      console.log(str);
+      // console.log(str);
       let updatedObj = notesObj.filter((note) => note.note !== str);
       notesObj = updatedObj;
     }
 
+    // check note
+    if (
+      target.classList.contains("notes__item-check") ||
```

Comment:
Problem 1: Checking if the click was on a checkbox or image
Original:
`if (target.classList.contains("notes__item-check") || target.tagName === "IMG")`

Why it's bad:
If you change the image to something else (like an SVG), it won't work anymore.

It's hard to know where exactly people can click.

Better way:
Wrap the checkbox and image in a container and use just one class on that container.

HTML:
````markdown
<div class="notes__item-check-container">
  <div class="notes__item-check"></div>
  <img src="check-icon.svg" />
</div>
````


JS:
```js
if (target.closest(".notes__item-check-container")) {
  // toggle check status
}